index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Awesome Books</title>
+</head>
+<body onload="displayCounterTime()">
```

Comment:
Great job @waseemqureshi2020 🤝

- [ ] [Optional]Kindly remove all Javascript from your HTML file. This is regarded as a bad practice to mix logic with the presentation markup. You may put all javascript code in a separate javascript file.